doc/source/tutorial/stats/outliers.md

```diff
@@ -0,0 +1,521 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": ["jupyterlite_sphinx_strip"]}
+
+```{eval-rst}
+.. notebooklite:: outliers.md
+   :new_tab: True
+```
+
+(outliers)=
+
++++
+
+# Trimming and Winsorization Transition Guide
+
+An outlier is an observation that differs substantially from other observations in a sample. There are many references available about identifying outliers, whether to make adjustments for outliers, and if so, what adjustments to make for outliers. This guide does not make any recommendations on these topics. Instead, the focus is how to perform two of the most common procedures:
+
+- trimming: removing outliers from the dataset, and
+- winsorization: replacing outliers with a more central value,
+
+using tools available in SciPy, with an emphasis on transitiong from use of legacy functions to the preferred approach.
+
+# Motivation
+In the past, SciPy has offered several "convenience functions" that combine trimming with computation of a statistic. Consider, for instance, `scipy.stats.trim_mean`.
+
+```{code-cell} ipython3
+import numpy as np
+from scipy import stats
+np.set_printoptions(linewidth=120)
+x = np.arange(20.)
+stats.trim_mean(x, proportiontocut=0.1)
+```
+
+Without carefully reading the documentation, it is not immediately obvious how the operation was performed. In this case, we would have gotten the same result whether `proportiontocut` refered to the proportion to cut from each tail or the total amount of data to remove.
+
+```{code-cell} ipython3
+# Remove 2/20 observations total before taking the `mean`, one in each tail
+np.mean(x[1:-1])
+```
+
+```{code-cell} ipython3
+# Remove 2/20 observations from *each* tail before taking the `mean`
+np.mean(x[2:-2])
+```
```

Comment:
I'm trying to show that it's not obvious what's really going on under the hood when the operations of trimming and taking a statistic are combined. In retrospect, I'm not sure if the particular approach was effective.

Reply:
Shall I leave the questions and remove the example code cells?